README.md

```diff
@@ -1,31 +1,56 @@
 # setup-cli
 
-This is a composite GitHub Action that sets up the Databricks CLI (preview version `>=0.100`) in your GitHub Actions workflow.
+Easily install the [Databricks CLI](https://github.com/databricks/cli) in your environment.
+
+Full documentation about installation can be found at:
+* (AWS) https://docs.databricks.com/en/dev-tools/cli/install.html
+* (Azure) https://learn.microsoft.com/en-us/azure/databricks/dev-tools/cli/install
+* (GCP) https://docs.gcp.databricks.com/en/dev-tools/cli/install.html
 
 ## Usage
 
-In your GitHub Actions workflow, use the following step:
+This repository contains an `install.sh` script that can be invoked to install the Databricks CLI.
+
+To install the latest version, run the following command:
+
+```bash
+curl -fsSL https://raw.githubusercontent.com/databricks/setup-cli/main/install.sh | sh
+```
+
+To install a specific version, you can replace `main` with a specific release tag:
+
+```bash
+curl -fsSL https://raw.githubusercontent.com/databricks/setup-cli/v0.221.1/install.sh | sh
+```
+
+## GitHub Actions
```

Comment:
This should probably point to https://learn.microsoft.com/en-us/azure/databricks/dev-tools/bundles/ci-cd and friends?

Reply:
Sure, good idea. That also talks about auth a bit.